frontend/src/components/MainGrid.jsx

```diff
@@ -30,9 +35,18 @@ export default class MainGrid extends React.Component {
   }
 
   // Update the state for a new country
-  handleCountryChange(iso_code) {
+  handleCountryChange(iso_code, selectedCaseNumber) {
```

Comment:
I wonder whether something like `currentTotalCases` would be clearer than `selectedCaseNumber`?

Reply:
Done!